rosetta/services/tx_operation.go

```diff
@@ -364,6 +450,92 @@ func getCrossShardSenderTransferNativeOperations(
 	}, nil
 }
 
+// delegator address => validator address => amount
+func getSideEffectOperationsFromUndelegationPayouts(
+	undelegationPayouts *hmy.UndelegationPayouts, opType string, startingOperationIndex *int64,
+) ([]*types.Operation, *types.Error) {
+	var opIndex int64
+	operations := []*types.Operation{}
+	if startingOperationIndex != nil {
+		opIndex = *startingOperationIndex
+	} else {
+		opIndex = 0
```

Comment:
`opIndex` is default to 0. Maybe this line is not necessary IMO. It can save 2 lines for the function ;)

Reply:
Do you see a reason to explicitly to assign the value 0?

Reply:
opps another one